### ISSUE TITLE
Auto Create an Empty Folder with Strange Filename, when ROM's dir containing Non-English Characters

### ISSUE THREAD

**Issue by eeuq7mi** [TASVideos/desmume#441]
Firstly, thanks for providing this excellent project!
Here I want to report a small problem. 

**OS:** Win 8.1 x64
**DeSmuME version:** 0.9.12 commit `4b77e9b`

# Reproduce
1. click: Menu -> Open ROM, use the "file browser dialog" to open the following path:  `D:\ROMs\にほんご\` (`にほんご` is a folder's name), and load a ROM under the folder `D:\ROMs\にほんご\a.nds`.

2. in `desmume.ini`:

the following content will be auto added:
```
[PathSettings]
Roms=D:\ROMs\にほんご
```

3. **Again**, click: Menu -> Open ROM. The "file browser dialog" will appear, auto located at the path:`D:\ROMs\にほんご\`.

And **at the same time**, an empty folder with Strange Filename `銇伝銈撱仈`, is auto created at `D:\ROMs\銇伝銈撱仈\`, (which is at the **parent dir** of `D:\ROMs\にほんご\`, i.e. `D:\ROMs\`).

---

Currently, no other negative effects is observed.

In summary, it will auto create an empty folder with strange filename, when ROM's **folder** containing Non-English Characters, for example, japanese characters.

# Workaround
1. Rename ROM's folder to pure english character only, e.g. `D:\ROMs\test\a.nds`.
2. Use Old emulator version "0.9.11", which doesn't have this bug.

**Comment by zeromus** [comment]
please stop using japanese characters, I am too busy to keep fixing these 10000s of bugs

**Comment by eeuq7mi** [comment]
**Bug fixed. Thanks a lot!**

DeSmuME is very popular in [non-english speaking countries](https://sourceforge.net/projects/desmume/files/stats/map).(Unofficial mirror is not counted in this statistics.) In these countries, it is very common to name the file/folder with non-english characters.

Someone  even translated DeSmuME into [Chinese](https://tieba.baidu.com/p/6600854282), [Japanese](http://enmakun.g1.xrea.com/emu/),etc. But the translators usually can't fix the bug.

Chinese Translation:
![chinese](https://user-images.githubusercontent.com/76865770/113798639-26f60000-9786-11eb-9355-a32b94e23e99.png)

Japanese Translation:
![japanese](https://user-images.githubusercontent.com/76865770/113798649-2c534a80-9786-11eb-8bde-905740c5a70d.jpg)

**Comment by Jules-A** [comment]
Maybe it would be easier to have an external library handle it?
